### ISSUE TITLE
App v 2 geofence support

### ISSUE THREAD

**Issue by irritanterik** [irritanterik/homey-tesla.com#13]
Geofence support is not (yet) available in the v2 Tesla App.
As an alternative for geofence 'Home' I added a capabillity (available as a token on the device) with name "Distance from Homey". But because of formatting this value with 'm' or 'km' this token cannot be used in conditions.
So some real geofence support should be added again to the v2 app. Starting by a simple 'Home' geofence with a fixed radius.

I didn't realize "afstand" was "afstand to homey". I find the naming confusing.

_Originally posted by @goyaweb in https://github.com/irritanterik/homey-tesla.com/issues/12#issuecomment-462374919_

**Comment by goyaweb** [comment]
Can you suggest a workaround to remove teh m or km in distance tag? Maybe using a js library?

**Comment by irritanterik** [comment]
Geofences will be back in 2.1.0! I'm porting the v1 settings screen to Homey V2 now. Also added a token with distance from Homey without km/m.

**Comment by Genghes** [comment]
NICE...  Now i HAVE to find your donation link.. :)

**Comment by gvdijnsen** [comment]
+1 for proper Geofencing. All my flows are seriously disabled ;-).

**Comment by goyaweb** [comment]
any updates on this one?

**Comment by irritanterik** [comment]
I submitted v 2.1.0 to the beta channel. You can download it from git or wait a few days for it in the app store. You'll need to remove and add your car again for the geofence capabillity to be available.

**Comment by goyaweb** [comment]
is there a way not to break all the flows when removing and re-adding the car as a device?

**Comment by irritanterik** [comment]
No, unfortunately not. But I expect this version is the last with breaking changes.

**Comment by glennvgastel** [comment]
Any chance you have some time to work on editing? 

I'm currently getting a `400` when the geofences are `PUT` to `https://{HOMEY_ID}.connect.athom.com/api/app/com.tesla/public/geofences`
The response is ;
```
{"code":500,"error":"invalid request data","error_description":"An unkown error has occured [unknown_error]"}
```

I've managed to work around it for now, but if you could pinpoint the problem, that would be nice :)
